### ISSUE TITLE
Add labels for form elements

### ISSUE THREAD

**Issue by nilshah98** [nilshah98/Portfolio#5]
**Issues :**
- There is a form element, to activate or deactivate night mode, but it does not have any label to support screenreaders or other accessibility tools

**Reference :**
- 
![image](https://user-images.githubusercontent.com/22821480/133277686-dc92fe39-94d9-4d10-8609-4028d76842f8.png)
- [Form element labels - web.dev](https://web.dev/label/?utm_source=lighthouse&utm_medium=lr)

**Acceptance Criteria :**
- [ ] Add label or disable that form element
